specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/0.1-preview/confidentialledger.json

```diff
@@ -0,0 +1,858 @@
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/MerkleProofElement"
+      }
+    },
+    "MerkleProofElement": {
+      "description": "An item in the Merkle proof.",
+      "type": "object",
+      "properties": {
+        "left": {
+          "type": "string"
+        },
+        "right": {
+          "type": "string"
+        }
+      }
+    },
+    "PagedLedgerEntries": {
+      "description": "Paginated ledger entries returned in response to a query.",
+      "type": "object",
+      "required": [
+        "state",
+        "entries"
+      ],
+      "properties": {
+        "state": {
+          "$ref": "#/definitions/LedgerQueryState"
+        },
+        "@nextLink": {
+          "description": "Path from which to retrieve the next page of results.",
+          "type": "string"
+        },
+        "entries": {
+          "$ref": "#/definitions/LedgerEntries"
+        }
+      }
+    },
+    "ReceiptContents": {
+      "description": "A receipt certifying the transaction at the specified id.",
+      "required": [
+        "leaf",
+        "nodeId",
+        "proof",
+        "root",
+        "signature"
+      ],
+      "type": "object",
+      "properties": {
+        "leaf": {
+          "description": "Merkle tree leaf for this transaction.",
+          "type": "string"
+        },
+        "nodeId": {
+          "description": "Id of the node returning the receipt.",
+          "$ref": "#/definitions/EntityId"
+        },
+        "proof": {
+          "description": "Merkle proof.",
+          "$ref": "#/definitions/MerkleProof"
+        },
+        "root": {
+          "description": "Root of the Merkle tree at the time the transaction was recorded.",
+          "type": "string"
+        },
+        "signature": {
+          "description": "Signature by the node, with its certificate, over the Merkle root.",
+          "type": "string"
+        }
+      }
+    },
+    "RoleAssignment": {
+      "description": "Object for assigning a role to a user.",
+      "required": [
+        "roleName"
+      ],
+      "properties": {
+        "roleName": {
+          "$ref": "#/definitions/LedgerUserRole"
+        },
+        "description": {
+          "description": "Description of the role.",
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "SubLedgerId": {
+      "description": "Identifier for sub-ledgers.",
+      "type": "string"
+    },
+    "TransactionId": {
+      "description": "A unique identifier assigned to each write transaction.",
+      "type": "string"
+    },
+    "TransactionReceipt": {
+      "description": "A receipt certifying the transaction at the specified id.",
+      "required": [
+        "state",
+        "transactionId"
+      ],
+      "properties": {
+        "receipt": {
+          "$ref": "#/definitions/ReceiptContents"
+        },
+        "state": {
+          "$ref": "#/definitions/LedgerQueryState"
```

Comment:
Is correct? The old PR didn't include this property and it seems to be related to `PagedLedgerEntries`

Reply:
Yep, it's correct - under the hood, retrieving receipts is a similar process to getting a transaction by a specific ID, thus requiring the addition of the `state` field.